Makefile.toml

```diff
@@ -284,3 +284,30 @@ run_task = "copy-lib"
 
 [tasks.post-android]
 dependencies = ["ffigen"]
+
+# TODO: non linux desktops
+# TODO: release build
+[tasks.desktop]
```

Comment:
if this is only linux, shouldn't we call it `linux` rather than desktop? windows and mac are both also desktop...

Reply:
Flutter linux support isn't stable as of now. Did the app run properly despite these errors?